src/components/Text/Link.js

```diff
@@ -1,38 +1,112 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-import { colors } from "../../theme";
+import Base from "./Base.styles";
 import StyledText from "./StyledText";
+import colors from "../../theme/colors";
 import getRelByTarget from "../../utils/link";
+import { getFontColor } from "../../utils/typography";
 
 export const LinkTitle = StyledText.withComponent(`a`).extend`
   color: ${({ color }) => color || colors.azure.base};
   text-decoration: none;
 `;
 
-const Link = ({ href, children, ...props }) => {
+const LinkBase = Base.withComponent("a").extend`
+  text-decoration: none;
+  transition: color 0.3s ease;
+  &:focus,
+  &:active,
+  &:visited,
+  &:hover {
+    color: ${props => getFontColor(props)};
```

Comment:
Is this per an aurora design or just to establish a baseline of styling for these states?

Reply:
This is mostly as a baseline so user agent styles don't take over the colors for the link.